.github/workflows/bundle.yml

```diff
@@ -0,0 +1,155 @@
+name: Bundle
+
+on:
+  push: # All branches, but appropriate paths only.
+    # branches:
+    #   - squeak-trunk
+      # - squeak-5.3
+      # - squeak-5.2
+    paths-ignore:
+      - '**.md' # Skip changes in documentation artifacts
+  pull_request:
+    branches:
+      - squeak-trunk
+      # - squeak-5.3
+      # - squeak-5.2
+    paths-ignore:
+      - '**.md'
+  schedule:
+    - cron: '0 0 * * *' # Build everyday at midnight
+  workflow_dispatch:
+
+jobs:
+  prepare-image:
+    strategy:
+      fail-fast: true
+      matrix:
+        smalltalk:
+          - Squeak-trunk
+          - Squeak64-trunk
+          # - Squeak-5.3
+          # - Squeak64-5.3
+          # - Squeak-5.2
+          # - Squeak64-5.2
+    runs-on: windows-latest
+    name: 🛠 Prepare image for ${{ matrix.smalltalk }}
+    env:
+      SMALLTALK_VERSION: ${{ matrix.smalltalk }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare image
+        shell: bash
+        run: ./prepare_image.sh
+        timeout-minutes: 20
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.smalltalk }}
+          path: |
+            tmp/*.sources
+            tmp/*.image
+            tmp/*.changes
+            tmp/version.sh
+            !tmp/Test*
```

Comment:
Just out of interest, where is the 90-day limit you mention indicated? :)

Reply:
In the repository settings. Only visible for admins.